Week2/homework/index.js

```diff
@@ -0,0 +1,117 @@
+            message: 'make your choice?',
+            choices: [options.first, options.second, options.third, options.fourth, options.fifth],
+        }, ]);
+
+        switch (userChoice.mainChoice) {
+            case options.first:
+                const userInput1 = await inquirer.prompt([
+                    { name: 'countryName', message: 'What is the country?' },
+                ]);
+                const query1 = `SELECT city.Name FROM country JOIN city on city.ID = country.Capital WHERE country.Name = ?`;
+                let result1 = await queryPromise(query1, userInput1.countryName);
+                if (result1.length === 0) {
+                    console.log(noMatch);
+                    break;
+                } else {
+                    console.log(`The capital is: ${result1[0].Name}`);
+                    break;
+                }
+
+            case options.second:
+                const userInput2 = await inquirer.prompt([
+                    { name: 'regionName', message: 'What is the region?' },
+                ]);
+                const query2 = `SELECT distinct Language FROM country join countrylanguage on country.Code=countrylanguage.CountryCode where Region =?`;
+                let result2 = await queryPromise(query2, userInput2.regionName);
+
+                if (result2.length === 0) {
+                    console.log(noMatch);
+                    break;
+                } else {
+                    console.log(`The languages are:`);
+                    result2.forEach(element => console.log(element.Language));
+                    break;
+                }
+
+            case options.third:
+                const userInput3 = await inquirer.prompt([
+                    { name: 'languageName', message: 'What is the language?' },
+                ]);
+                const query3 = `SELECT  count(city.name) as Cities FROM city join countrylanguage on city.CountryCode=countrylanguage.CountryCode where Language=? `;
+                let result3 = await queryPromise(query3, userInput3.languageName);
+                if (result3[0].Cities === 0) {
+                    console.log(noMatch);
+                    break;
+                } else {
+                    console.log(`The number of cities is: ${result3[0].Cities}`);
+                    break;
+                }
+            case options.fourth:
+                const userInput4 = await inquirer.prompt([
+                    { name: 'regionName', message: 'What is the region?' },
+                    { name: 'languageName', message: 'What is the language?' },
+                ]);
+                const query4 = `SELECT Name FROM country join countrylanguage on country.Code=countrylanguage.CountryCode where Region =? and Language=? and IsOfficial='T'; `;
+                let result4 = await queryPromise(query4, [userInput4.regionName, userInput4.languageName]);
+                if (result4.length === 0) {
+                    console.log(noMatch);
```

Comment:
Here it should be false but it's fine that way ( false because the user could enter two good input but still no countries speak this language).

Reply:
Thanks for accepting that. 
If I am the user , then the word "False"  is meaningless to  mee. I might think there is something wrong in the program.  maybe it is not meaningless for the one who requested it.